data/hooks/conf_regen/15-nginx

```diff
@@ -47,6 +47,12 @@ do_pre_regen() {
 
   # install / update plain conf files
   cp plain/* "$nginx_conf_dir"
+  # remove the panel overlay if this is specified in settings
+  panel_overlay=$(yunohost settings get 'panel_overlay.enabled')
+  if [ $panel_overlay = false ] || [ $panel_overlay = False ]
+  then
+    echo "#" > "${nginx_conf_dir}/yunohost_panel.conf.inc"
```

Comment:
There is an issue if i set it to False and next to True, the overlay is still hidden :/

Reply:
I think I had this problem but if you clear the cache of the browser after setting it to True, it may work

Reply:
It was a mistake from me, i read you code and don't understand it correctly. So it's solved